src/usr/local/www/system_camanager.php

```diff
@@ -308,6 +317,7 @@
 
 		if (!$input_errors) {
 			write_config();
+			ca_trustedcerts_generate();
```

Comment:
I agree.  No need for insults or name-calling.

Where did I insult you?  Where did I call you a name?

You certainly hurled some insults at me though.

My way is no more riskier than how the CA bundle is currently being handled.  And like I stated.  Rewriting files at boot up is still not the "right" way to ensure file integrity.  There are better, more reliable methods with far less overhead.

Leaving unverified data in place because the filename happens to be correct isn't "well engineered".

But you and pfSense are okay with that as long is it's other just as critical files.  Yes.  That logic is well reasoned out.  Not.

I have never said they shouldn't be verified.  But if they are going to be then do it "right".  With a well engineer method and consistently apply it to critical files.